package.json

```diff
@@ -83,8 +84,8 @@
     "stylelint-config-prettier": "^9.0.4",
     "stylelint-config-standard": "^23.0.0",
     "stylelint-order": "^5.0.0",
-    "xml2js": "^0.5.0",
-    "typescript": "^5.0.2"
+    "typescript": "^5.0.2",
```

Comment:
This change will fall off after rebasing once [this PR](https://github.com/pinterest/gestalt/pull/2932) is merged